src/transformers/models/big_bird/configuration_big_bird.py

```diff
@@ -0,0 +1,162 @@
+        num_hidden_layers (:obj:`int`, `optional`, defaults to 12):
+            Number of hidden layers in the Transformer encoder.
+        num_attention_heads (:obj:`int`, `optional`, defaults to 12):
+            Number of attention heads for each attention layer in the Transformer encoder.
+        intermediate_size (:obj:`int`, `optional`, defaults to 3072):
+            Dimensionality of the "intermediate" (i.e., feed-forward) layer in the Transformer encoder.
+        hidden_act (:obj:`str` or :obj:`function`, `optional`, defaults to :obj:`"gelu_fast"`):
+            The non-linear activation function (function or string) in the encoder and pooler. If string,
+            :obj:`"gelu"`, :obj:`"gelu_fast"`, :obj:`"relu"`, :obj:`"selu"` and :obj:`"gelu_new"` are supported.
+        hidden_dropout_prob (:obj:`float`, `optional`, defaults to 0.1):
+            The dropout probabilitiy for all fully connected layers in the embeddings, encoder, and pooler.
+        attention_probs_dropout_prob (:obj:`float`, `optional`, defaults to 0.1):
+            The dropout ratio for the attention probabilities.
+        max_position_embeddings (:obj:`int`, `optional`, defaults to 4096):
+            The maximum sequence length that this model might ever be used with. Typically set this to something large
+            just in case (e.g., 1024 or 2048 or 4096).
+        type_vocab_size (:obj:`int`, `optional`, defaults to 2):
+            The vocabulary size of the :obj:`token_type_ids` passed when calling :class:`~transformers.BigBirdModel` or
+            :class:`~transformers.TFBigBirdModel`.
+        initializer_range (:obj:`float`, `optional`, defaults to 0.02):
+            The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
+        layer_norm_eps (:obj:`float`, `optional`, defaults to 1e-12):
+            The epsilon used by the layer normalization layers.
+        use_cache (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether or not the model should return the last key/values attentions (not used by all models). Only
+            relevant if ``config.is_decoder=True``.
+        attention_type (:obj:`str`, `optional`, defaults to :obj:`block_sparse`)
+            Whether to use block sparse attention (with n time) as introduced in paper or original attention layer
+            (with n^2 time). Possible values are `original_full` & `block_sparse`.
+        use_bias (:obj:`bool`, `optional`, defaults to :obj:`True`)
+            Whether to use bias in query, key, value.
+        rescale_embeddings (:obj:`bool`, `optional`, defaults to :obj:`False`)
+            Whether to rescale embeddings with (hidden_size ** 0.5)
+        block_size (:obj:`int`, `optional`, defaults to :obj:`64`)
+            Size of each block. Useful only when `attention_type` is `block_sparse`.
+        num_random_blocks (:obj:`int`, `optional`, defaults to :obj:`3`)
```

Comment:
```suggestion
        num_random_blocks (:obj:`int`, `optional`, defaults to 3)
```

we just leave numbers as they are :-)